src/css/components/panel.scss

```diff
@@ -27,6 +27,19 @@ $panel-padding: $grid-3;
     margin-bottom: $grid-1;
   }
 
+  .route-form {
+    background-color: $color-lightestgray;
+
+    label {
+      color: $color-blue;
+    }
+
+    .route-form-preview {
+      font-size: inherit;
+      padding-left: 0;
+    }
+  }
+
```

Comment:
Technically we shouldn't nest the route-form classes under panel, but we also don't have a place for the route form code, it currently resides in the dashboard CSS code. I guess I'm OK with this as long as we put a TODO about how this is temporary. Maybe like `TODO: remove once we abstract route form into own component`?